django/cantusdb_project/main_app/templates/chant_detail.html

```diff
@@ -268,7 +268,7 @@ <h4 id="concordances">List of concordances</h4>
 
                 <h4>List of melodies</h4>
                 <span id="melodyLoadingPrompt" style="display: none; color: #922"><b>Loading melodies...</b></span>
-                <a id="melodyButton" href="#" onclick="loadMelodies('{{ chant.cantus_id }}'); return false;">» Display the melodies connected with this chant</a>
+                <a id="melodyButton" href="#" onclick="loadMelodies('{{ chant.cantus_id }}'); return false;">Display the melodies connected with this chant</a>
```

Comment:
I notice that the new class ("guillemet") is not applied here. This also isn't included in the screenshots, so of course your intention could be to not have the double "greater than sign" (lol, what is the English name for these?) here. So I'm approving since you will know whether this was oversight or intentional! :)

Reply:
I think its's "guillemet" even in English - at least, wikipedia provides a few anglicized pronunciations of it.

I consciously did not apply `class="guillemet"` here because this is in the main panel of the page and not in the sidebar. These show up in a couple of other places and I'm not sure whether we'll want them all to be `display: block`.

But in general, I'm not sure why OldCantus uses guillemets for some links and not others. It doesn't seem to indicate that they're external web pages. So if we were to adopt a consistent approach, I'm not sure what the correct consistent approach would be...